plugins/hypervisors/xenserver/test/com/cloud/hypervisor/xenserver/resource/wrapper/xenbase/CitrixRequestWrapperTest.java

```diff
@@ -436,7 +436,7 @@ public void testResizeVolumeCommand() {
         final Answer answer = wrapper.execute(resizeCommand, citrixResourceBase);
         verify(citrixResourceBase, times(1)).getConnection();
 
-        assertFalse(answer.getResult());
+        //assertFalse(answer.getResult());
```

Comment:
@priyankparihar @anshul1886 Is this commented statement required for future reference? If not, better to remove this line.